bin/storj.js

```diff
@@ -36,6 +36,8 @@ program._storj.loglevel = function() {
 };
 
 program._storj.PrivateClient = function(options) {
+  options.blacklistFolder = DATADIR;
```

Comment:
Instead of supplying a folder to write a file into, what do you think about just supplying a file path?